homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -220,63 +219,32 @@ def __init__(self, name, device, entry, unique_id, attribute, coordinator):
         """Initialize the entity."""
         super().__init__(name, device, entry, unique_id, coordinator)
 
-        self._name = name
+        self._description = SENSOR_TYPES[attribute]
```

Comment:
Could we use something like `self._sensorconfig` instead of `self._description`?

Reply:
Sure, I'll change the name tomorrow.